raiden/transfer/token_network.py

```diff
@@ -105,15 +107,25 @@ def handle_channelnew(
     ] = (our_address, partner_address)
 
     if our_address not in token_network_state.channelidentifiers_to_channels:
-        token_network_state.channelidentifiers_to_channels[our_address]: Dict[AddressHex, NettingChannelState] = dict()
+        token_network_state.channelidentifiers_to_channels[our_address] = dict()
 
     # Ignore duplicated channelnew events. For this to work properly on channel
     # reopens the blockchain events ChannelSettled and ChannelOpened must be
     # processed in correct order, this should be guaranteed by the filters in
     # the ethereum node
     if channel_identifier not in token_network_state.channelidentifiers_to_channels[our_address]:
+
         token_network_state.channelidentifiers_to_channels[our_address][channel_identifier] = channel_state
 
+        #if channel_state.test: ## mismo hub, 2 light clients
```

Comment:
This must be changed in order to do it only when both participants are handled lc